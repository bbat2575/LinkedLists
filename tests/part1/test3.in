NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 0
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 5
1
2.2
2e4
@
No soup for you!
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
NEW 26
a
b
c
d
e
f
g
h
i
j
k
l
m
n
o
p
q
r
s
t
u
v
w
x
y
z
