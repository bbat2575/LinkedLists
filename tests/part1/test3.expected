
Type HELP for a list of options
Press <CTRL + D> to Quit

> List 0: 

> List 1: 

> List 2: 

> List 3: 

> List 4: 

> List 5: 

> List 6: 

> List 7: 

> List 8: 

> List 9: 

> List 10: 

> List 11: 

> List 12: 

> List 13: 

> List 14: 

> List 15: 

> List 16: 

> List 17: 

> List 18: 

> List 19: 

> List 20: 

> List 21: 

> List 22: 

> List 23: 

> List 24: 

> List 25: 

> List 26: 

> List 27: 

> List 28: 

> List 29: 

> List 30: 

> List 31: 

> List 32: 

> List 33: 

> List 34: 

> List 35: 

> List 36: 

> List 37: 

> List 38: 

> List 39: 

> List 40: 

> List 41: 

> List 42: 

> List 43: 

> List 44: 

> List 45: 

> List 46: 

> List 47: 

> List 48: 

> List 49: 

> List 50: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 51: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 52: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 53: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 54: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 55: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 56: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 57: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 58: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 59: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 60: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 61: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 62: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 63: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 64: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 65: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 66: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 67: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 68: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 69: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 70: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 71: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 72: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 73: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 74: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 75: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 76: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 77: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 78: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 79: 1 -> 2.20 -> 20000.00 -> @ -> No soup for you!

> List 80: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 81: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 82: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 83: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 84: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 85: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 86: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 87: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 88: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 89: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 90: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 91: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 92: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 93: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 94: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 95: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 96: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 97: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 98: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> List 99: a -> b -> c -> d -> e -> f -> g -> h -> i -> j -> k -> l -> m -> n -> o -> p -> q -> r -> s -> t -> u -> v -> w -> x -> y -> z

> 
